### ISSUE TITLE
what is vlive and poetry

### ISSUE THREAD

**Issue by journeytosilius** [kvdomingo/pygfycat#1]
```
# pip
pip install vlive

# poetry
poetry add vlive
```

what's this and why do I have to install it ? It's not enough with installing the wrapper ?

**Comment by journeytosilius** [comment]
also, I am  trying to use this and it doesn't work by passing the gfycat url to gpycat.get_gfycat():

original url : `https://gfycat.com/BriskInsistentHeifer`

`item = gpycat.get_gfycat("BriskInsistentHeifer")`

```
pydantic.error_wrappers.ValidationError: 8 validation errors for GfyItem
content_urls -> mobile
  field required (type=value_error.missing)
content_urls -> mobilePoster
  field required (type=value_error.missing)
content_urls -> mp4
  field required (type=value_error.missing)
content_urls -> webm
  field required (type=value_error.missing)
content_urls -> webp
  field required (type=value_error.missing)
userData
  field required (type=value_error.missing)
userDisplayName
  field required (type=value_error.missing)
userProfileImageUrl
  field required (type=value_error.missing)
connection closed
```

**Comment by kvdomingo** [comment]
> ```
> # pip
> pip install vlive
> 
> # poetry
> poetry add vlive
> ```
> 
> what's this and why do I have to install it ? It's not enough with installing the wrapper ?

Sorry about the `vlive` part, forgot to change it from the template I used 😅

`poetry` is just another package manager; you don't need to install it if you prefer to use `pip` or some other package manager.

**Comment by kvdomingo** [comment]
> also, I am trying to use this and it doesn't work by passing the gfycat url to gpycat.get_gfycat():
> 
> original url : `https://gfycat.com/BriskInsistentHeifer`
> 
> `item = gpycat.get_gfycat("BriskInsistentHeifer")`
> 
> ```
> pydantic.error_wrappers.ValidationError: 8 validation errors for GfyItem
> content_urls -> mobile
>   field required (type=value_error.missing)
> content_urls -> mobilePoster
>   field required (type=value_error.missing)
> content_urls -> mp4
>   field required (type=value_error.missing)
> content_urls -> webm
>   field required (type=value_error.missing)
> content_urls -> webp
>   field required (type=value_error.missing)
> userData
>   field required (type=value_error.missing)
> userDisplayName
>   field required (type=value_error.missing)
> userProfileImageUrl
>   field required (type=value_error.missing)
> connection closed
> ```

This should be fixed by #2. Just run `pip install -U gpycat` to get the updates. The gfycat API docs don't really make it clear which fields are optional.
